docs/trellis/master/python.md

```diff
@@ -0,0 +1,122 @@
+using trellis-cli to make your life easier.
+
+When dealing with Trellis and Python, there's three key points:
+
+1. Make sure you have a stable version of Python and pip installed
+2. Use [trellis-cli](https://github.com/roots/trellis-cli) since it handles
+   dependencies for you
+3. **Never** use `sudo` when installing packages with `pip`
+
+## Python 2 vs Python 3
+
+While Python 2 reached end-of-life in 2019, it's still supported in Trellis for
+now. However, we do *recommend using Python 3* whenever possible. And since
+Python 3 is mostly the default version now, there's not many reasons to still
+use Python 2.
+
+Unlike most languages that have a single version installed at a time, and only
+offer an "unversioned" single binary path (such as just `node`), Python can be
+more confusing because some operating systems treat them separately.
+
+Regardless of the OS, it's still possible to symlink `python3` to `python` for
+convenience as well.
+
+## Installing Python
+
+### macOS
+
+Newer versions of macOS like Monterey and Big Sur come with both versions 2 and
+3. Annoyingly though, the unversioned `python` is 2.7x while `python3` needs to
+be explicitly used for Python 3.
+
+While using the system Python on macOS should work fine, the main downside is
+that the versions are only updated when macOS itself has a new major version.
+
+If you want to have more control over Python versions, we recommend using a tool
+like [pyenv](https://github.com/pyenv/pyenv) or [asdf](https://github.com/danhper/asdf-python)
+to install specific versions globally (or even per project if that's needed).
+
+We **do not recommend** installing Python from Homebrew. This might be
+surprising since it goes against most guides and recommendations but we believe
+using Python from Homebrew will cause more problems long-term due to its newer 
+"feature" of auto-upgrading packaages as described in [this article](https://justinmayer.com/posts/homebrew-python-is-not-for-you/).
+
+After Python is installed and working, you'll also need to ensure pip is installed. If `pip` or `pip3` does not exist, it can be installed like this:
+
+```bash
+sudo easy_install pip
+```
+
+### Ubuntu
+
+Ubuntu 20.04 comes default with Python 3 available as `python3`
+only. There's no "unversioned" `python`. If you want to install Python 2, it
+will be installed as `python2`.
+
+The [`python-is-python3`])(https://packages.ubuntu.com/focal/python-is-python3) package
+exists solely as an easy to symlinks `/usr/bin/python` to `python3`.
+
+```bash
+sudo apt-get install -y python3 python-is-python3 python3-pip
+```
+
+### Windows
+
+Roots [recommends using WSL](https://roots.io/docs/getting-started/windows/)
+when using Windows. Since WSL uses Ubuntu, the [section](#ubuntu) above applies
+but you might need to also install `python3` (from within WSL):
+
+```bash
+sudo apt-get install -y python3 python-is-python3 python3-pip
+```
```

Comment:
This is identical to the command above so "but you might need to also install `python3` (from within WSL)" is confusing (both commands install python3).

Reply:
Ah I updated it and forgot about this. Removed, thanks